charts/seed-monitoring/charts/prometheus/rules/apiserver-connectivity-check.rules.yaml

```diff
@@ -0,0 +1,33 @@
+groups:
+- name: apiserver-connectivity-check.rules
+  rules:
+  - alert: TcpConnectionTimeoutToApiServer
+    expr: probe_http_duration_seconds{job=~"blackbox-exporter-kube-api-probe-[0-9]", phase="connect"} >= 128
+    labels:
+      service: apiserver-connectivity-check
+      severity: warning
+      type: shoot
+    annotations:
+      summary: A TCP Connection to the API server timed out.
+      description: At least one probe failed to establish a connection in under 128 seconds to the API server via the infrastructure load balancer.
+
+  - alert: SlowConnectionToApiServer
+    expr: count(probe_http_duration_seconds{phase="connect",job=~"blackbox-exporter-kube-api-probe-[0-9]"} > 0.01)/count(probe_http_duration_seconds{phase="connect",job=~"blackbox-exporter-kube-api-probe-[0-9]"})
```

Comment:
Is the query of this alert correct? Think the condition is missing.
`count of slow probes / count of all probes > 0.1`

Just a question. Should we really start with only 10%? We execute ten probes in parallel and if only two probes need more than 10ms to respond then we already fire the alert. I'm not sure how to tune the settings here wisely to avoid false positives.